pyseir/inference/infer_rt.py

```diff
@@ -37,7 +38,7 @@ class RtInferenceEngine:
     ref_date:
         Reference date to compute from.
     confidence_intervals: list(float)
-        Confidence interval to compute. 0.90 would be 90% credible
+        Confidence interval to compute. 0.95 would be 90% credible
```

Comment:
95%

Reply:
It is correct as is.  This is the upper bound to compute which provides a 90% confidence.